check.js

```diff
@@ -714,6 +715,49 @@ async function csrCanForfeit(contracts, batch, address) {
   return res;
 }
 
+/**
+ * Check if the address can give invites to point
+ * @param {Number} point - Point number to give invites to
+ * @param {String} address - The caller address
+ * @return {Promise<Object>} A result and reason pair
+ */
+async function canSetPoolSize(contracts, point, address) {
+  return checkActivePointOwner(contracts, azimuth.getPrefix(point), address);
+}
+
+/**
+ * Check if as can send point as an invite to to
+ * @param {Number} as - The point that would send the invite
+ * @param {Number} point - The point that would be sent as invite
+ * @param {String} to - The Ethereum address recipient of the invite
+ * @param {String} address - The caller address
+ * @return {Promise<Object>} A result and reason pair
+ */
+async function canSendInvitePoint(contracts, as, point, to, address) {
+  let res = await checkActivePointOwner(contracts, as, address);
+  if (!res.result) {
+    return res;
+  }
+  res.result = false;
+  if (to === address) {
```

Comment:
IIRC there's an `addressEquals` function in utils that makes sure these are checksummed before comparing them.  Worth using here?